website/content/docs/upgrading/instructions/upgrade-to-1-10-x.mdx

```diff
@@ -0,0 +1,140 @@
+compatible with v1.10 and will cause issues during the upgrade that could result in
+agents failing to start.
+
+## Requirements
+
+- All Consul servers, clients and snapshot agents should be on a version of Consul >= 1.8.0 and < 1.10.0.
+
+## Assumptions
+
+This guides makes the following assumptions:
+
+- You are familiar with the [General Upgrade Process](/docs/upgrading/instructions/general-process).
+- You have the ability to run Consul CLI commands. 
+
+## Considerations
+
+The main breaking changes in Consul Enterprise 1.10 that needs special handling during upgrading are
+the licensing changes outlined on the [Specific Version Details](/docs/upgrading/upgrade-specific#consul-1-10-0)
+page. There are notes on that paged about other changes that might cause issues during an upgrade as well.
+You can find more granular details in the full [changelog](https://github.com/hashicorp/consul/blob/release/release/1.10.0/CHANGELOG.md#1100-june-22-2021).
+Looking through these changes prior to upgrading is highly recommended.
+
+## Procedures
+
+**1.** Retrieve the datacenter's current license by executing the following CLI command:
+
+```shell
+consul license get -signed
+```
+
+This will be useful to have on hand when preparing the license for the 1.10 upgrades later
+on in the process.
+
+**2.** Determine if ACLs are enabled by running the following CLI command.
+
+```shell
+consul info | grep "acl ="
+````
+
+If you receive output like either of the following then ACLs are enabled:
+
+```text
+Error querying agent: Unexpected response code: 403 (Permission denied)
+```
+
+```text
+        acl = enabled
+
+```
+
+<Tabs>
+<Tab heading="ACLs Enabled">
+
+**3.** Upgrade all server agents to the latest 1.8.x or 1.9.x release.
+
+**4.** Upgrade all client agents to the latest 1.8.x or 1.9.x release.
+
+**5.** Upgrade all snapshot agents to the latest 1.8.x or 1.9.x release.
+
+**6.** Pre-configure the server agents with a license.
+
+This can either be set with the `license_path` configuration item, the 
+`CONSUL_LICENSE_PATH` environment variable or the `CONSUL_LICENSE` 
+environment variable. See the [licensing documentation](/docs/enterprise#licensing) 
+for more information about how to configure the license.
+
+**7.** Upgrade all server agents to v1.10.0
+
+**8.** Upgrade all client agents to v1.10.0
+
+**9.** Upgrade all snapshot agents to v1.10.0
+
+</Tab>
+<Tab heading="ACLs Disabled">
+
+**3.** Upgrade all server agents to the latest 1.8.x or 1.9.x release.
+
+**4.** Pre-configure the client agents with a license.
+
+This can either be set with the `license_path` configuration item, the 
+`CONSUL_LICENSE_PATH` environment variable or the `CONSUL_LICENSE` 
+environment variable. See the [licensing documentation](/docs/enterprise#licensing) 
+for more information about how to configure the license.
+
+**5.** Upgrade all client agents to the latest 1.8.x or 1.9.x release.
```

Comment:
feels worth calling out the normal processes for checking breaking changes and taking snapshots etc must be followed here?

Reply:
Specifically here? We call out checking for breaking changes in the Considerations section already.

I can add the bit about snapshotting but where should it be. I could see us advocating for taking a snapshot at each point where versions are stable (pre-upgrade and 1.8.13/1.9.7 upgrade is complete across servers, clients and snapshot agents)

Reply:
Given our conversation I think we decided it is okay as is after my modifications to add in information about when to take snapshots.